EDU-9108 - Updating README.md (#47)
vtex-apps/product-comparison: docs/README.md

  ...
```

> > ℹ️ *The `store.search.product-comparison` blocks will wraps the `store.search` block with comparison context. The replacement is needed so we can synchronously display the selected products in the Product Comparison drawer.*

Reviewer: ```suggestion
> ℹ️ The `store.search.product-comparison` blocks wrap the `store.search` block with comparison context. The replacement is needed so it can synchronously display the selected products in the Product Comparison drawer.
```
